lib/rules/web/admin_console/4.8/catalog.xyaml

```diff
@@ -11,6 +11,7 @@ create_app_from_imagestream:
   action: goto_create_app_from_imagestream_page
   action: wait_form_loaded
   action: click_try_sample
+  action: click_try_sample
```

Comment:
workaround the issue describe in bug https://bugzilla.redhat.com/show_bug.cgi?id=1979801 , before the fix the error is:
```
07-07 10:26:42.754        [02:26:08] INFO> found 1 element elements with selector: {:xpath=>"//button[contains(text(),'Create')]"}
07-07 10:26:42.754        [02:26:38] ERROR> #<Watir::Exception::ObjectDisabledException: element present, but timed out after 30 seconds, waiting for #<Watir::Button: located: true; {:xpath=>"//button[contains(text(),'Create')]", :index=>0, :tag_name=>"button"}> to be enabled>
```

Reply:
@XiyunZhao  Please help review, thanks!